department-of-veterans-affairs/caseflow
# Div/appeals 57463

Resolves [Correspondence: Intake Existing Appeals table doesn't show all information for Appeals](https://jira.devops.va.gov/browse/APPEALS-57463)

# Description
Added missing columns to the serializer function

## Acceptance Criteria

## Testing Plan
1. Login in as an Inbound Ops User or Superuser
2. Queue
3. Go to Your Correspondences
4. Work a Correspondence through Review Package
5. Create Intake from Review Package
6. On Step 2  Review Tasks & Appeals=
      1. Click related to an existing appeal
      2. Observe that the following columns should appear 
            1. Status
            2. Decision Date
            3. Assigned to


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
